src/App.tsx

```diff
@@ -1,15 +1,17 @@
 import React from "react";
 
 import "./app.scss";
+import Footer from "./components/Footer";
 
+import Header from "./components/Header";
 import Router from "./router";
-import Navigation from "./components/Navigation";
 
 const App: React.FC = (): JSX.Element => {
   return (
     <div className="app">
```

Comment:
Do you need this div with class ?

Reply:
this class gives 100vh heihgt for all content